doc/nrf/ug_tfm.rst

```diff
@@ -48,18 +52,26 @@ When building for ``nrf9160dk_nrf9160ns``, UART1 must be disabled in the non-sec
 Otherwise, the non-secure application will fail to run.
 The recommended way to do this is to copy the .overlay file from the :ref:`tfm_hello_world` sample.
 
+Using TF-M with nrf_security
+============================
+
+When using the :ref:`nrfxlib:nrf_security`, if :option:`CONFIG_BUILD_WITH_TFM` is enabled together with :option:`CONFIG_NORDIC_SECURITY_BACKEND`, the TF-M secure image will enable the use of the hardware acceleration of Arm CryptoCell.
+
+In such case, the Kconfig configurations in the Nordic Security Backend control the features enabled through TF-M.
```

Comment:
@hakonfam anything to add to this vis-á-vis configurations for enabled services?

Reply:
@b-gent  It is possible to configure what crypto modules to include in TF-M by using the kconfig options found here: https://github.com/nrfconnect/sdk-zephyr/blob/master/modules/trusted-firmware-m/Kconfig.tfm.crypto_modules

These configurations should be set so that they match the configuration of nrf_security.

Reply:
It's probably worth mentioning HW unique keys here as well.

> TF-M utilizes HW Unique Keys <reference to hw_unique_key.rst> when the PSA Crypto key derivation APIs are used.

Reply:
@oyvindronningstad  @hakonfam 

The enabled services you are mentioning, are they specific to nrf_security? This section is called 'Using TF-M with nrf_security'.

Reply:
Not specific, but highly related to nrf_security. They use nrf_cc3xx_platform APIs.

Reply:
The enabled services are specific to TF-M. The functionality inside the secure image is not one-to-one with the API that you provide to the outside world (i.e. exposed secure services).

Reply:
When it comes to Øyvinds line, I think it is not accurate enough. There are key derivation APIs exposed as part of the PSA Crypto APIs that doesn't rely on HW unique keys

Also, the nrf_cc3xx_platform is not part of nrf_security. This is a regular library with platform-specific functions to utilize CryptoCell features. 

I think only the PSA PS (Protected Storage) make use of the HW unique keys by default in TF-M. This is not part of the PSA Crypto APIs (or PSA Crypto service)

Reply:
Using the psa_crypto key_derivation APIs from NS will utilize HUKs if `TFM_CRYPTO_ALG_HUK_DERIVATION` is specified as the alg.

Reply:
Let's sum up what needs to be done here because I am getting lost.

1. The current name of this section is 'Using TF-M with nrf_security'. Maybe we should change that to something else? Something around 'Enabling secure services'? I just feel all the discussion here is about much more than just nrf_security.

2. I will add a sentence 'You can configure what crypto modules to include in TF-M by using the kconfig options found here:" that was suggested at the beginning by Hakon.

3. Then, add a sentence 'TF-M utilizes HW Unique Keys when the PSA Crypto key derivation APIs are used.' with a link to the HUK library doc.

@frkv  @oyvindronningstad  let me know if this is ok, I need to finalize this asap